pkg/build/strategies/sti/sti.go

```diff
@@ -218,6 +219,17 @@ func (b *STI) PostExecute(containerID, location string) error {
 
 	buildEnv := append(scripts.ConvertEnvironment(env), b.generateConfigEnv()...)
 
+	labels := map[string]string{}
+	if b.sourceInfo != nil {
+		labels = map[string]string{
+			"io.openshift.build.commit.author":     b.sourceInfo.Author,
```

Comment:
I've talked with @bparees about that, we've agreed to have the prefix configurable. By default it'll be `io.s2i...` and in OpenShift it'll be `io.openshift...`